Releases.md

```diff
@@ -487,7 +487,7 @@ Changes in std version 0.59.0:
 - fix(std/node): fix readFile types, add encoding types (#6451)
 - fix(std/node): global process should usable (#6392)
 - fix(std/node/process): env, argv exports (#6455)
-- fix(std/testing) assertArrayContains should work with any array-like (#6402)
+- fix(std/testing) assertArrayIncludes should work with any array-like (#6402)
```

Comment:
There should be no changes to this file, revert

Reply:
O shoot, I thought I excluded that file. Reverted.